althea_kernel_interface/src/interface_tools.rs

```diff
@@ -231,6 +231,19 @@ impl dyn KernelInterface {
             Ok(())
         }
     }
+
+    /// Gets the network namespace name that holds the thread this function was called from
+    pub fn get_namespace(&self) -> Result<String, Error> {
+        let output = self.run_command("ip", &["netns", "identify"]).unwrap();
+        match String::from_utf8(output.stdout) {
```

Comment:
what happens if we run this in the default namespace? Does it return an empty string? 

Also what happens if we run it and ip netns is not installed? We should special case that and return the default namespace.  Right now it will panic on line 238 it doesn't make a huge amount of sense to handle the panic for interpreting the result as utf8 but not handling the error for if the command fails.